misc/acrn-config/board_config/new_board_kconfig.py

```diff
@@ -41,7 +41,7 @@ def get_addr_for_hv(ram_range, hpa_size):
     tmp_order_key = sorted(ram_range)
     for start_addr in tmp_order_key:
         mem_range = ram_range[start_addr]
-        if mem_range > int(hpa_size, 16):
+        if mem_range > int(hpa_size, 10):
```

Comment:
Could you align the format of the address/size as hex?

Reply:
"mem_range" is the 10 base int,  "hpa_size" is the 16 base.  Here is only for internal compare,  For output to the c source file, they are all 16 base hex.  i think keeps the 10 base here for compare is ok.